### PR TITLE
Set property fields to mandatory

### LINKED ISSUE #500: Make appropriate Property fields required in database
**Purpose**
Ensure Property fields Name, Address, City, State and Zip cannot be null in the database.

**Proposed Solution**
Add `nullable=False` to these fields.
Can add tests to Schema if you're up to it in `tests/schema/test_property_schema.py`

**Acceptance Criteria**
- [ ] All tests are still passing

### PR DESCRIPTION
**Pull request by Chris-Boe**
### What issue is this solving?
Closes codeforpdx/dwellingly-app/issues/500

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? n
Any new dependencies to install? n
Any special requirements to test? n

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

File: tests/schemas/test_property_schema.py [hunk 1]
```diff
@@ -24,6 +24,21 @@ def test_valid_payload(self, create_property_manager):
         no_validation_errors = {}
         assert no_validation_errors == PropertySchema().validate(valid_payload)
 
+    def test_missing_required_parameters(self, create_property_manager):
+        invalid_payload = {
+            "created_at": Time.one_year_from_now(),
+            "updated_at": Time.today(),
```
Comment (aedwardg) [inline tests/schemas/test_property_schema.py]: Let's get rid of `created_at` and `updated_at` here and also in the test above it (`test_valid_payload`).  These fields are created by the database and shouldn't be included in the payload.  Plus, using `Time.one_year_from_now()` isn't appropriate here (`created_at` shouldn't be one year after `updated_at` 🤔).  It must have slipped in with a previous PR.

Once those are removed, the `utils.time` import won't be necessary either, so let's get rid of that as well.  

Other than that, this test looks good 👍

**Review by aedwardg — CHANGES_REQUESTED** [review]
Thanks for tackling this, @Chris-Boe! It looks good, just a couple quick changes before it's merge-ready.

File: tests/schemas/test_property_schema.py (hunk 1 above)
Comment (Chris-Boe) [reply tests/schemas/test_property_schema.py]: Ah yeah I just copied and pasted, whoops! I'll update those

**Review by aedwardg — APPROVED** [review]
LGTM 👍

**Pull request merged**